config/environments/development.rb

```diff
@@ -36,4 +36,5 @@
   config.action_controller.action_on_unpermitted_parameters = :raise
 
   config.action_mailer.default_url_options = { host: "hours.dev" }
+  config.action_dispatch.tld_length = 0
```

Comment:
Maybe set this to the following since most of us use `subdomain.hours.local` as dev url
```rb
ENV["TLD_LENGTH"] || 1
```